example/main.rs

```diff
@@ -96,6 +122,64 @@ fn create_txt_record<ApiClientType: ApiClient>(
     print_response(response);
 }
 
+fn list_routes<ApiClientType: ApiClient>(arg_matches: &ArgMatches, api_client: &ApiClientType) {
+    let usage = "usage: list_routes ZONE_ID";
+
+    let zone_id_missing = format!("missing '{}': {}", "ZONE_ID", usage);
+    let zone_identifier = arg_matches
+        .value_of("zone_identifier")
+        .expect(&zone_id_missing);
+
+    let response = api_client.request(&workers::ListRoutes { zone_identifier });
+
+    print_response_json(response);
+}
+
+fn create_route<ApiClientType: ApiClient>(arg_matches: &ArgMatches, api_client: &ApiClientType) {
+    let usage = "usage: create_route ZONE_ID SCRIPT_NAME ROUTE_PATTERN";
+
+    let zone_id_missing = format!("missing '{}': {}", "ZONE_ID", usage);
+    let zone_identifier = arg_matches
+        .value_of("zone_identifier")
+        .expect(&zone_id_missing);
+
+    let route_pattern_missing = format!("missing '{}': {}", "ROUTE_PATTERN", usage);
+    let route_pattern = arg_matches
+        .value_of("route_pattern")
+        .expect(&route_pattern_missing);
+
+    let script_name = arg_matches.value_of("script_name");
+
+    let response = api_client.request(&workers::CreateRoute {
+        zone_identifier,
+        params: workers::CreateRouteParams {
+            pattern: route_pattern.to_string(),
+            script: script_name.map(|n| n.to_string()),
+        },
+    });
+
+    print_response_json(response);
+}
+
+fn delete_route<ApiClientType: ApiClient>(arg_matches: &ArgMatches, api_client: &ApiClientType) {
+    let usage = "usage: delete_route ZONE_ID ROUTE_ID";
+
+    let zone_id_missing = format!("missing '{}': {}", "ZONE_ID", usage);
+    let zone_identifier = arg_matches
+        .value_of("zone_identifier")
+        .expect(&zone_id_missing);
+
+    let route_id_missing = format!("missing '{}': {}", "ROUTE_PATTERN", usage);
+    let route_id = arg_matches.value_of("route_id").expect(&route_id_missing);
```

Comment:
Does this line actually work? The registered clap arguments below seem to refer to a `route_identifier` instead of a `route_id`

Reply:
hrm. thought i ran a manual test for delete but it seems like i did not. will fix and re-do tests.